.github/workflows/build.yml

```diff
@@ -0,0 +1,190 @@
+
+            -
+                name: Shutdown default MySQL
+                run: sudo service mysql stop
+
+            -
+                name: Setup MySQL
+                uses: mirromutth/mysql-action@v1.1
+                with:
+                    mysql version: "${{ matrix.mysql }}"
+                    mysql root password: "root"
+
+            -
+                name: Output PHP version for Symfony CLI
+                run: php -v | head -n 1 | awk '{ print $2 }' > .php-version
+
+            -
+                name: Install certificates
+                run: symfony server:ca:install
+
+            -
+                name: Run Chrome Headless
+                run: google-chrome-stable --enable-automation --disable-background-networking --no-default-browser-check --no-first-run --disable-popup-blocking --disable-default-apps --allow-insecure-localhost --disable-translate --disable-extensions --no-sandbox --enable-features=Metal --headless --remote-debugging-port=9222 --window-size=2880,1800 --proxy-server='direct://' --proxy-bypass-list='*' http://127.0.0.1 > /dev/null 2>&1 &
+
+            -
+                name: Run webserver
+                run: (cd tests/Application && symfony server:start --port=8080 --dir=public --daemon)
+
+            -
+                name: Get Composer cache directory
+                id: composer-cache
+                run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            -
+                name: Cache Composer
+                uses: actions/cache@v2
+                with:
+                    path: ${{ steps.composer-cache.outputs.dir }}
+                    key: ${{ runner.os }}-php-${{ matrix.php }}-composer-${{ hashFiles('**/composer.json **/composer.lock') }}
+                    restore-keys: |
+                        ${{ runner.os }}-php-${{ matrix.php }}-composer-
+
+            -
+                name: Prepare migrations for Sylius 1.7
+                if: matrix.sylius == '1.7.*'
+                run: |
+                    cp etc/sylius17/migrations/* tests/Application/src/Migrations/
+                    cp src/Migrations/* tests/Application/src/Migrations/
+                    sed -i -- 's/Sylius\\RefundPlugin\\Migrations/DoctrineMigrations/g' tests/Application/src/Migrations/*
+                    cp etc/sylius17/bundles.php tests/Application/config/
+                    cp etc/sylius17/doctrine_migrations.yaml tests/Application/config/packages/
+                    cp etc/sylius17/security.yaml tests/Application/config/packages/
+                    rm tests/Application/config/packages//lexik_jwt_authentication.yaml
+                    rm tests/Application/config/routes/sylius_api.yaml
+
+            -
+                name: Prepare migrations for Sylius 1.8
+                if: matrix.sylius == '1.8.*'
+                run: |
+                    composer require sylius-labs/doctrine-migrations-extra-bundle --no-interaction --no-update
```

Comment:
I guess that this is not needed as you are creating a test database schema by `doctrine:schema:create` instead of using migrations